scripts/gluster-block/gluster-block-basic.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# if anything fails, we'll abort
+set -e
+
+# install basic dependencies for building the tarball and srpm
+yum -y install git autoconf automake gcc libtool bison flex make mock
+# gluster repositories contain additional -devel packages
+yum -y install centos-release-gluster
+yum -y install libuuid-devel targetcli tcmu-runner glusterfs-api-devel
```

Comment:
Is the tcmu-runner from centos-base enough? Or is there one in the Storage SIG?

Reply:
It was enough on fedora! I haven't technically tried the same on centos... 

@nixpanic @pkalever anything you are aware of?

Reply:
Looks reasonable, but there is no need to install mock if you wont use it.

Reply:
Also, tcmu-runner would get pulled from the Storage SIG because centos-release-gluster gets installed on the line above.

Reply:
On the other hand, it may not be enough. tcmu-runner did not have a recent release, and there are a few changes that gluster-block needs. although those could be runtime only, like new targetcli features. @pkalever would know the details.

Reply:
@pkalever need your feedback on this one!

Reply:
@amarts @nixpanic, is it a best practice to build the packages on our own at our nightly builds, instead of waiting for the respective maintainers for updating the packages.

I'm not sure about centos, but fedora dependencies are in testing phase:
https://bodhi.fedoraproject.org/updates/python-configshell-1.1.fb25-2.fc28%20python-rtslib-2.1.fb69-1.fc28%20targetcli-2.1.fb49-1.fc28#comment-853188

Also there is a new tcmu-runner release 1.4.0, but Maurizio is trying hard to take over the maintainer ship of the fedora packaging and will help us with updating the package at F28 and F29.

Gluster-block is currently blocked on https://github.com/gluster/gluster-block/pull/57#issuecomment-434582261, for its 0.4 release, request @nixpanic to help there.

Thanks!

Reply:
Looking at the dependency of tcmu-runner, I am inclined to do a nightly build so that we don't have to install too many packages everytime. I am now thinking then, if we should build it, and run tests once a day to start with.

I will send another PR to do a nightly, that way, we can take a decision on this, later

Reply:
Changed this to run nightly, with tcmu-runner also considered from nightly.